===================================================================
RCS file: /cvs/gnome/gtk+/gdk/x11/gdkwindow-x11.c,v
retrieving revision 1.247
retrieving revision 1.248
diff -c -r1.247 -r1.248
*** gdk/x11/gdkwindow-x11.c	2005/08/12 13:13:54	1.247
--- gdk/x11/gdkwindow-x11.c	2005/08/15 12:35:04	1.248
***************
*** 2913,2919 ****
    g_return_if_fail (pixmap == NULL || !parent_relative);
    g_return_if_fail (pixmap == NULL || gdk_drawable_get_depth (window) == gdk_drawable_get_depth (pixmap));
  
!   if (!gdk_drawable_get_colormap (pixmap))
      {
        g_warning ("gdk_window_set_back_pixmap(): pixmap must have a colormap");
        return;
--- 2913,2919 ----
    g_return_if_fail (pixmap == NULL || !parent_relative);
    g_return_if_fail (pixmap == NULL || gdk_drawable_get_depth (window) == gdk_drawable_get_depth (pixmap));
  
!   if (pixmap && !gdk_drawable_get_colormap (pixmap))
      {
        g_warning ("gdk_window_set_back_pixmap(): pixmap must have a colormap");
        return;
